tests/transforms/test_qcut.py

```diff
@@ -66,7 +66,7 @@
     qml.CNOT(wires=[2, 3])
     qml.RY(0.543, wires=2)
     qml.RZ(0.876, wires=3)
-    qml.expval(qml.PauliZ(wires=[0]))
+    qml.expval(qml.PauliZ(wires=[0]) @ qml.PauliZ(wires=[3]))
```

Comment:
Needed, because our current standard test case actually is an example of the edge case - i.e., with disconnected components.